TheJumpCloud/support
# check nuget before runasuser

## Issues
* [SA-3135](https://jumpcloud.atlassian.net/browse/SA-3135) - Check for Nuget before installing `runAsUser`

## What does this solve?

Before installing a module Nuget must be installed on a windows device. This is typically installed at some point but not by default. 

I also bumped the `net-ldap` version due to a [dependabot issue raised this week](https://github.com/TheJumpCloud/support/security/dependabot/1)

## Is there anything particularly tricky?

## How should this be tested?

On a new system or a system without Nuget/ runasuser installed ([instructions to uninstall nuget](https://stackoverflow.com/a/71315138)) run the command changed in this PR, the command should install Nuget, RunAsUser and execute the code through the JumpCloud PowerShell Module

## Screenshots


[SA-3135]: https://jumpcloud.atlassian.net/browse/SA-3135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
